rest-api-spec/src/main/resources/rest-api-spec/api/update_by_query.json

```diff
@@ -149,7 +149,7 @@
         },
         "scroll_size": {
           "type": "number",
-          "defaut_value": 100,
+          "default_value": 100,
```

Comment:
Same (note: I drop by word family, so by default all fixes to `default` will get dropped at the same time)